### DISCUSSION TITLE
So is Devkit just Userkit but you have to compile it?

### DISCUSSION THREAD

**Discussion by winbamstudios** [CosmosOS/Cosmos#2399] [General]
i wanna know

**Comment by ascpixi** [comment]
The devkit is the "bleeding edge" version of Cosmos; it has more features but may be less stable than the userkit. Because of the nature of the devkit, it requires you to compile it yourself.

**Reply by winbamstudios** [reply]
ok

**Comment by zarlo** [comment]
in shot yes
